db/src/users.rs

```diff
@@ -29,7 +29,8 @@ pub async fn get_user(pool: &PgPool) -> Result<Vec<User>> {
 
 pub async fn get_username_by_id(pool: &PgPool, user_id: Uuid) -> Result<String> {
     let record = sqlx::query!("SELECT user_name FROM Users WHERE user_id = $1", user_id)
-        .fetch_optional(pool)
+        .fetch_one(pool)
         .await?;
-    Ok(record.map(|r| r.user_name).unwrap_or_default())
+
+    Ok(record.user_name)
```

Comment:
The update was to handle from `Option` type to `String` type returned by the `fetch_one` method.

Reply:
Ah I think this change for the `Option` probably wanted to stay so I reverted it!

This option wasnt really about the null-ness of the Password hash, its the difference between the `fetch_once` and `fetch_optional`, and the usages of this method in the CLI already handled the option, so I left it as is!